.github/workflows/odh-notebooks-sync.yml

```
@@ -0,0 +1,82 @@
+  PYTHON_VERSION: ${{ github.event.inputs.python-version }}
+  ODH_JUPYTER_TRASH_CLEANUP_RELEASE_VERSION: ${{ github.event.inputs.odh_jupyter_trash_cleanup_release_version }}
+  UPDATER_BRANCH: odh-sync-updater-${{ github.run_id }}
+  UPSTREAM_OWNER: ${{ github.event.inputs.upstream-repository-organization }}
+  REPO_OWNER: ${{ github.event.inputs.repository-organization }}
+  REPO_NAME: ${{ github.event.inputs.repository-name }}
+  GITHUB_TOKEN: ${{ secrets.ODH_JUPYTER_TRASH_CLEANUP_MACHINE_ACCOUNT_TOKEN }}
+  MINIMUM_SUPPORTED_PYTHON_VERSION: 3.11
+
+jobs:
+  build:
+    runs-on: ubuntu-latest-8core
```

actionlint (1.7.7)

label "ubuntu-latest-8core" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)